docs/network-flow-visibility.md

```diff
@@ -142,26 +163,23 @@ the flow. All the IEs used by the Antrea Flow Exporter are listed below:
 
 #### Types of Flows and Associated Information
 
-Currently, the Flow Exporter feature provides visibility for Pod-to-Pod, Pod-to-Node,
-Node-to-Pod, Node-to-Node and Pod-to-Service network flows along with the associated
-statistics such as data throughput (bits per second), packet throughput (packets
-per second), cumulative byte count, cumulative packet count etc. Pod-To-Service
-flow visibility is supported only [when Antrea Proxy enabled](feature-gates.md).
+Currently, the Flow Exporter feature provides visibility for Pod-to-Pod, Pod-to-Service
+and Pod-to-External network flows along with the associated statistics such as data
+throughput (bits per second), packet throughput (packets per second), cumulative byte
+count, cumulative packet count etc. Pod-To-Service flow visibility is supported
+only [when Antrea Proxy enabled](feature-gates.md). In the future, we will extend
```

Comment:
is enabled

Reply:
not addressed yet

maybe specify that this is now the default:  `Pod-To-Service flow visibility is supported
only [when Antrea Proxy is enabled](feature-gates.md), which is the case by default starting with Antrea v0.11`.